setup.py

```diff
@@ -19,8 +19,10 @@
     "beautifulsoup4>=4.4.1",
     "celery==3.1.10",
     "contextdecorator==0.10.0",
+    "django-enumfield==1.2.1",
     "django-betty-cropper>=0.2.6",
     "django-filter==0.9.2",
+    "django-jsonfield==0.9.19",
```

Comment:
```https://github.com/theonion/django-bulbs/blob/master/bulbs/ads/models.py#L7```

Uses the previous one that doesn't support `jsonb` I suggest we migrate that one to the newer package in another PR, since historically 3rd party field packages have been not terrific at tricky migrations.